test/jdk/tools/jlink/plugins/CompressorPluginTest.java

```diff
@@ -146,6 +146,65 @@ public void test() throws Exception {
                 new ResourceDecompressorFactory[]{
                     new ZipDecompressorFactory(),
                 }, Collections.singletonList(".*Exception.class"));
+
+        // compress level zip-0 == no compression
+        Properties optionsZip0 = new Properties();
+        DefaultCompressPlugin compressPluginZip0 = new DefaultCompressPlugin();
+        options0.setProperty(compressPluginZip0.getName(), "zip-0");
```

Comment:
I suspect this supposed to be `optionsZip0.setProperty(....)` instead of `options0.setProperty(....)`?

Reply:
I think, there's also a typo in the next line:
```
checkCompress(classes, compressPlugin,
```
should have been:
```
checkCompress(classes, compressPluginZip0,
```
I haven't fully looked at the existing test code in `checkCompress` method, but it might need a closer look as to why this test passed. I would have expected it to fail due to the typos.

Reply:
Thanks all!